src/osip-monitoring-view/src/test/java/edu/kit/pse/osip/monitoring/view/dialogs/DialogTest.java

```diff
@@ -0,0 +1,49 @@
+package edu.kit.pse.osip.monitoring.view.dialogs;
+
+import javafx.application.Platform;
+import javafx.stage.Stage;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Test class for the Dialog classes.
+ * 
+ * @author Martin Armbruster
+ * @version 1.0
+ */
+public class DialogTest extends ApplicationTest {
+    /**
+     * Saves a test instance of the AboutDialog.
+     */
+    private AboutDialog about;
+    /**
+     * Saves a test instance of the HelpDialog.
+     */
+    private HelpDialog help;
+    
+    @Override
+    public void start(Stage arg0) throws Exception {
+        about = new AboutDialog();
+        help = new HelpDialog();
+    }
+    
+    /**
+     * Tests the constructors of the Dialog classes.
+     * 
+     * @throws Exception when something goes wrong.
+     */
+    @Test
+    public void testDialogs() throws Exception {
+        assertNotNull(about);
+        assertNotNull(help);
+        Platform.runLater(() -> {
+            about.show();
+            help.show();
+            about.hide();
+            help.hide();
+        });
```

Comment:
Tests finden nicht im Application Thread statt, oder? Sonst könnte man sich das runLater sparen. Ich glaub bei Exceptions in einem anderen Thread läuft nämlich der Test trotzdem durch